# BrushedMotorFeedbackMsg
uint8 part_id
uint8 id
uint8 current
uint16 rpm

# See definitions in BrushedMotor.h
